instrumentation/sidekiq/lib/opentelemetry/instrumentation/sidekiq/instrumentation.rb

```diff
@@ -27,6 +27,12 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
           gem_version >= MINIMUM_VERSION
         end
 
+        option :enable_job_class_span_names, default: false, validate: ->(v) { v.is_a?(TrueClass) || v.is_a?(FalseClass) }
```

Comment:
Is this sufficient or do we need to explicitly test the class?
```suggestion
        option :enable_job_class_span_names, default: false, validate: ->(v) { v == true || v == false }
```

Reply:
I don't think we need to test the class.  I'll update it.